sputnikdao2/src/proposals.rs

```diff
@@ -2,9 +2,10 @@ use std::collections::HashMap;
 use std::convert::TryFrom;
 
 use near_contract_standards::fungible_token::core_impl::ext_fungible_token;
+use near_contract_standards::storage_management::StorageBalance;
```

Comment:
Thanks for the effort on the SDK side here, @constantindogaru https://github.com/near/near-sdk-rs/pull/630

Given some delays I anticipate in getting this change pushed to a prerelease or release for the SDK crate, can be please duplicate this struct in this project instead of using the `near_contract_standards` crate? Let's add a note in there about the PR for us in the future, like:

```rust
// Struct duplicated while pending: https://github.com/near/near-sdk-rs/pull/630
#[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize)]
#[serde(crate = "near_sdk::serde")]
pub struct StorageBalance {
    pub total: U128,
    pub available: U128,
}
```

(And then import any Borsh or serde we need to)

I hate to be hacky, but I think this is best to unblock us as I'm aware of issues the SDK team are dealing with that will likely delay this effort.